R/dettl_runner.R

```diff
@@ -30,20 +36,20 @@ new_import <- function(path, db_name = NULL) {
 #' This runs the full extract, transform, test and load steps or runs specific
 #' stages specified by 'run_stages'.
 #'
-#' @param import The dataImport object.
+#' @param import The DataImport object.
 #' @param run_stages Which stages should be run in the import process. Runs all
 #' if NULL.
 #' @param dry_run If TRUE then any database changes are rolled back when the
 #' import completes. i.e. the load stage can be run and tests executed but the
 #' db will be rolled back.
 #'
-#' @return The processed dataImport object.
+#' @return The processed DataImport object.
 #' @export
 #'
 run_import <- function(import, run_stages = NULL, dry_run = FALSE) {
-  if (is.null(import) || class(import) != "dataImport") {
+  if (is.null(import) || class(import) != "DataImport") {
```

Comment:
as above, `class(import) != "DataImport"` should be `!inherits(import, "DataImport")`